### ISSUE TITLE
Could not resolve host: yum$ociregion.oracle.com

### ISSUE THREAD

**Issue by bernermic** [oracle/container-images#27]
Installing new packages (here example: openssl) via microdnf fails with `Could not resolve host: yum$ociregion.oracle.com` when doing: `microdnf update && microdnf install openssl`

Installation of packages works fine when I just use: `microdnf install openssl` 

How to test failure:
```
$ docker run -it --rm oraclelinux:8-slim bash
$ microdnf update && microdnf install openssl
```

In the update log I can see this entry:
```
...
Updating: oraclelinux-release-el8;1.0-14.el8;x86_64;ol8_baseos_latest
/var/tmp/rpm-tmp.hpfrdH: line 4: /etc/yum/vars/ociregion: No such file or directory
...
```

### Expected: 
installing packages is possible after a microdnf update

**Comment by Djelibeybi** [comment]
Thanks, we've just logged this internally as well but glad for the report.

**Comment by Djelibeybi** [comment]
FYI, you can workaround this issue by running:

```
# mkdir -p /etc/dnf/vars
# echo "" > /etc/dnf/vars/ociregion
```

**Comment by Djelibeybi** [comment]
This has been resolved with the release of [`libdnf-0.48.0-5.0.2`](https://linux.oracle.com/errata/ELBA-2020-5931.html). Thanks again for the report.
